src/woo_publications/publications/api/serializers.py

```diff
@@ -171,6 +180,39 @@ def validate_publicatiestatus(
 
         return value
 
+    def get_or_create_organisation_member(self, owner: EigenaarSerializer | None):
+        identifier = self.context["request"].headers[AUDIT_USER_ID_PARAMETER.name]
+        naam = self.context["request"].headers[AUDIT_USER_REPRESENTATION_PARAMETER.name]
+
+        if owner:
+            identifier = owner["identifier"]
+            naam = owner["naam"]
+
+        org_member, _ = OrganisationMember.objects.get_or_create(
+            identifier=identifier,
+            naam=naam,
+        )
```

Comment:
here too I'd use `update_or_create`

identifiers should stay the same for a given person, but their (display) name can change due to a variety of reasons